dashboard-app/src/util/dataManipulation/aggregatedToTableData.ts

```diff
@@ -47,12 +50,37 @@ const abbreviateMonths = evolve({
   ),
 });
 
-export const aggregatedToTableData = ({ title, data, sortBy }: Params) => {
+const round = MathUtil.roundTo(2);
+
+
+const toUnitDuration = (unit: DurationUnitEnum, duration: Duration.Duration) => {
+  switch (unit){
+    case DurationUnitEnum.DAYS:
+      return round(Duration.toWorkingDays(duration));
+
+    case DurationUnitEnum.HOURS:
+      return round(Duration.toHours(duration));
+  }
+};
+
+const add = (a: number, b: any) => a + (isNaN(b) ? 0 : Number(b));
+
+export const calculateTotalsUsing = (unit: DurationUnitEnum) => evolve({
```

Comment:
Can we pull this function out of this file? it's being used by csv manipulation & `aggregatedToTableData`

Reply:
Do you mean the `Using`?

Reply:
Yup

Reply:
I don't think the functionality is clear from the naming. This function also does more than calculate, it also adds these values to the data structure. Approving as keen to get this in, but would be up for a rename in future.